Userland/Libraries/LibGUI/GML/Parser.cpp

```diff
@@ -131,4 +134,201 @@ ErrorOr<NonnullRefPtr<GMLFile>> parse_gml(StringView string)
     return file;
 }
 
+enum class UseObjectConstructor : bool {
```

Comment:
All of this hardly belongs into Parser.cpp :)

Reply:
Since it's not generally useful for anything else depending on LibGUI, I'd put this into GMLCompiler directly.